controllers/kafkauser_controller.go

```diff
@@ -238,7 +238,13 @@ func (r *KafkaUserReconciler) Reconcile(ctx context.Context, request reconcile.R
 				return requeueWithError(reqLogger, "failed to reconcile user secret", err)
 			}
 		}
-		kafkaUser = user.DN()
+		kafkaUser, err = user.DN()
```

Comment:
**Note**: we need to rename that function for public interface clarity purposes....